src/main/java/com/ifactory/press/db/solr/search/SafariBlockJoinQuery.java

```diff
@@ -183,56 +192,6 @@ int getParentDoc() {
             return parentDoc;
         }
 
-        @Override
-        public int nextDoc() throws IOException {
```

Comment:
I assume we got rid of these method because we're now using `<scorer>.iterator()` for things like `nextDoc()` and `advance()`?

Reply:
Are we losing any important custom behavior by using `<scorer>.iterator()`?